setup.py

```diff
@@ -0,0 +1,38 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
```

Comment:
Regarding pypi credentials. I hope you will continue to look after the project as you have got more understanding and visibility to what is going on with the metrix++ tool and python world. If there is a way to share an access with me to the  python packages repository, please share it with me just for safety / security / back up purposes. (Let me know what I should do, so you can share it with me). I am not intending to publish metrixpp package myself, but who knows when and if you need help... It is good that you have automated it (as far as I understand).

Reply:
Just create an account on pypi and tell me your username. I will then share the project with you.
https://pypi.org/account/register/